build/build.sh

```diff
@@ -0,0 +1,15 @@
+export http_proxy_host=$(echo $http_proxy | awk -F':' {'print $2'} | tr -d '/')
+export https_proxy_host=$(echo $https_proxy | awk -F':' {'print $2'} | tr -d '/')
+export http_proxy_port=$(echo $http_proxy | awk -F':' {'print $3'} | tr -d '/')
+export https_proxy_port=$(echo $http_proxy | awk -F':' {'print $3'} | tr -d '/')
+
+if [ "$use_remote" = "1" ]; 
+    then 
+        cd /tmp/
+        git clone https://github.com/secure-device-onboard/iot-platform-sdk 
+        cd /tmp/iot-platform-sdk/ 
+        mvn clean install -Dhttp.proxyHost=$http_proxy_host -Dhttp.proxyPort=$http_proxy_port -Dhttps.proxyHost=$https_proxy_host -Dhttps.proxyPort=$http_proxy_port 
+        cd demo && find . -name \*.war -exec cp --parents {} /home/sdouser/iot-platform-sdk/demo \;      
+    else 
+        mvn clean install -Dhttp.proxyHost=$http_proxy_host -Dhttp.proxyPort=$http_proxy_port -Dhttps.proxyHost=$https_proxy_host -Dhttps.proxyPort=$http_proxy_port ;
```

Comment:
Another nitpick.

Define a variable for maven proxy flags like

_JAVA_OPTIONS="-Dhttp.proxyHost=$http_proxy_host -Dhttp.proxyPort=$http_proxy_port -Dhttps.proxyHost=$https_proxy_host -Dhttps.proxyPort=$http_proxy_port"

And then use 'mvn clean install' (as it would automatically pick up _JAVA_OPTIONS).

Reply:
Sure.